commcare_connect/templates/tailwind/pages/opportunities_list.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'tailwind/base.html' %}
```

Comment:
nit: template name change (opportunity -> opportunities) seems unnecessary

Reply:
Agreed, I will try to be wary of these for next pages